content/release-10-16-0/announcements-10-16-0.md

```diff
@@ -12,6 +12,16 @@ layout: bundle
 
 ### Security changes
 
+##### Deprecation of SMS TFA feature
+
+The SMS TFA (Two-Factor Authentication) feature is deprecated. With release 10.17+, it will be removed and not be functional any longer, and we will no longer support SMS TFA.
```

Comment:
```suggestion
As announced with [release 10.15](/release-10-15-0/announcements-10-15-0), the SMS TFA (Two-Factor Authentication) feature is deprecated. With release 10.17+, it will be removed and not be functional any longer, and we will no longer support SMS TFA.
```